charts/tfy-configs/templates/cicd-templates-configmap.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-cicd-templates-cm
```

Comment:
I need help with this
Should I add `{{ .Release.Name }}` or not ?

Reply:
this is fine